internal/cmd/agent/deployer/deployer.go

```diff
@@ -184,9 +184,9 @@ func addLabelsFromOptions(nsLabels map[string]string, optLabels map[string]strin
 	}
 
 	// Delete labels not defined in the options.
-	// Keep the name label as it is added by helm when creating the namespace.
+	// Keep the corev1.LabelMetadataName label as it is added by kubernetes when creating the namespace.
```

Comment:
Function-level doc (L180) should be updated too ;)

Reply:
oh true :smile: changed in https://github.com/rancher/fleet/pull/2009/commits/316ff02e4258c9c538417c85eb2e124e5ce3dfb1